upgrade/scripts/k8s/upgrade_control_plane.sh

```diff
@@ -65,34 +65,34 @@ masters=$(grep -oP 'ncn-m\d+' /etc/hosts | sort -u)
 k8sVersionUpgradeTo=$(kubeadm version -o json | jq -r '.clientVersion.gitVersion')
 
 for master in $masters; do
-  echo "Upgrading kube-system pods for $master:"
+  echo "DEBUG Upgrading kube-system pods for $master:"
   echo ""
   pdsh -b -S -w $master "kubeadm upgrade apply ${k8sVersionUpgradeTo} -y"
   rc=$?
   if [ "$rc" -ne 0 ]; then
     echo ""
-    echo "ERROR: The 'kubeadm upgrade apply' failed. The output from this script should be inspected"
-    echo "       and addressed before moving on with the upgrade. If unable to determine the issue"
-    echo "       and run this script without errors, discontinue the upgrade and contact HPE Service"
-    echo "       for support."
+    echo "ERROR The 'kubeadm upgrade apply' failed. The output from this script should be inspected"
```

Comment:
This is better, but because these lines may be mixed with lines for other messages it would be better if the entire error message was one line.